clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java

```diff
@@ -145,49 +141,6 @@ void testAuthBasic(Function<HttpClientBuilder, HttpClientBuilder> config) throws
                     UTF_8));
   }
 
-  @Test
-  void testTracing() throws Exception {
```

Comment:
where are we testing this now, given that we're removing it here?

Reply:
In the class you mentioned in the above comment

Reply:
yeah I got confused by what is where, so def. makes sense to clean this up here